supplier_management_site/auth/eventbrite_ar.py

```diff
@@ -0,0 +1,25 @@
+from django.views.i18n import get_formats
+from social_core.backends.eventbrite import EventbriteOAuth2
+from django.conf import settings
+from django.views import i18n
+from django.utils.translation import to_locale, get_language
+
+
+class EventbriteAuth2(EventbriteOAuth2):
+
+    def authorization_url(self):
+        if to_locale(get_language()) == 'es':
+            self.AUTHORIZATION_URL = 'https://www.eventbrite.com.ar/oauth/authorize'
```

Comment:
You may want to resolve the tld on a separate function and append the rest of the URI to it. That way when you add additional tlds to your tld resolver (or if you want to change how to resolve tlds), you just have to modify one function. Unit testing would be easier too since it decouples testing of the tld resolution logic from the other 3 functions.

Reply:
Done :)